presto-parser/src/main/java/com/facebook/presto/sql/parser/SqlParser.java

```diff
@@ -233,6 +237,7 @@ public void exitNonReserved(SqlBaseParser.NonReservedContext context)
             context.getParent().removeLastChild();
 
             Token token = (Token) context.getChild(0).getPayload();
+            warningConsumer.accept(new ParsingWarning(format("Non-reserved word used: %s", token.getText()), token.getLine(), token.getCharPositionInLine()));
```

Comment:
We don't want to report every non reserved word. We should only report those non-reserved words which are reserved. Currently it is only the `CURRENT_ROLE` keyword